smoke-tests/designer/features/pageobjects/pages/formRunner.page.js

```diff
@@ -14,28 +49,54 @@ class FormRunnerPage {
     return browser.$(".govuk-textarea");
   }
 
+  /**
+   * Selects a value from a list
+   * @param listValue
+   */
+  selectFromList(listValue) {
+    return browser.$(".govuk-select").selectByVisibleText(listValue);
+  }
+
   /**
    * Select radio using part of its name and then clicks continue
    * @param radioName
    */
-  selectRadio(radioName) {
+  selectRadio(radioName, save = true) {
     this.fieldSet.waitForDisplayed();
     browser
       .$$(".govuk-radios__label")
       .find((el) => el.getText().includes(radioName))
       .click();
-    this.continueButton.click();
+    if (save) {
+      this.continueButton.click();
+    }
+  }
+
+  /**
+   * Used to input the date for a date parts field
+   * @param day
+   * @param month
+   * @param year
+   */
+  inputDate(day, month, year) {
+    this.inputField("Day", day);
+    this.inputField("Month", month);
+    this.inputField("Year", year);
   }
 
   get continueButton() {
+    return browser.$("#submit");
+  }
+
+  get submitButton() {
     return browser.$(".govuk-button");
   }
 
   /**
    * Inputs text into a multiline text component and then clicks continue
    * @param text
    */
-  multilineText(text) {
+  textBox(text) {
```

Comment:
And this one too actually, didn't spot it until I'd added the other comment

Reply:
@gstevenson fixed